SPECS-EXTENDED/xerces-c/xerces-c.spec

```diff
@@ -0,0 +1,260 @@
+Summary:        Validating XML Parser
+Name:           xerces-c
```

Comment:
We already have this:
- 
https://github.com/microsoft/CBL-Mariner/pull/5976

Reply:
I know, it was not merged back then. I will rebase and remove this.